demos/python_demos/speech_recognition_demo/utils/context_timer.py

```diff
@@ -0,0 +1,12 @@
+import time
+
+
+class Timer:
+    def __enter__(self):
+        self.elapsed = None
+        self.start = time.time()
```

Comment:
You should be using `time.perf_counter()` instead.

Reply:
Please use timeit.default_timer() it is a canonical way

Reply:
Done, with timeit..